live/sources/String+PrintWithColor.swift

```diff
@@ -29,6 +29,7 @@ extension String {
      *  - Parameter color: The color to print the string using
      */
     func print(withColor color: AnsiColor) {
-        Swift.print(color.rawValue + self)
+        /// Append a white color code to the string, to reset the console color
+        Swift.print(color.rawValue + self + AnsiColor.white.rawValue)
```

Comment:
Ah, that sounds a lot better! 👍 We actually don't need a case for it, we can just inline the string at the end of the `print` call.